README.md

```diff
@@ -114,45 +114,15 @@ Version: 3
 
 </details>
 
-### Google Play Store **(Deprecated)**
+### Google Play Store **(Experimental branch)**
```

Comment:
It is not a branch in this repo, as an alternative I propose `Experimental app by @fornwall`

Reply:
Yes, but that's mainly not to clutter this repo's issues with the PS-specific questions, I'd imagine. The app is just a branch off our older app code that @fornwall plans to integrate back eventually.

I'll let you two hash out this title.